basxconnect/core/layouts/settings_layout.py

```diff
@@ -74,6 +74,25 @@ def personsettings(request):
     return ret
 
 
+def maintenancesettings(request):
+    ret = layout.grid.Grid(R(C(hg.H3(_("Maintenance")))), gutter=False)
+    ret.append(
+        R(
+            C(
+                layout.button.Button.fromlink(
+                    Link(
+                        reverse("core.vocabulary.browse"),
+                        label=_(Vocabulary._meta.verbose_name_plural),
+                    ),
+                    icon="edit",
+                )
+            ),
+            style="margin-bottom: 2rem",
+        )
+    )
```

Comment:
You can leave this empty, I will assign you some things to add here in one of the next tasks.